backend/db_schema/CJP_Schema.sql

```diff
@@ -0,0 +1,100 @@
+CREATE TYPE "skill_level" AS ENUM (
+  'beginner',
+  'intermediate',
+  'expert',
+  'advanced'
+);
+
+CREATE TABLE "jobpost" (
+  "id" SERIAL PRIMARY KEY,
+  "role" varchar,
+  "location" varchar,
+  "tags" varchar,
+  "description" varchar,
+  "exp_required" int,
+  "company_name" int,
+  "link_to_apply" varchar,
+  "email" varchar
+);
+
+CREATE TABLE "jobpost_skill" (
+  "id" SERIAL PRIMARY KEY,
+  "jobpost_id" int,
+  "skill_id" int
+);
+
+CREATE TABLE "user" (
+  "id" SERIAL PRIMARY KEY,
+  "first_name" varchar,
+  "last_name" varchar,
+  "email" varchar,
+  "is_admin" boolean,
+  "user_type" varchar,
+  "phone_no" int,
+  "gender" varchar,
+  "heighest_education" varchar,
+  "created_at" timestamp,
+  "current_ctc" int,
+  "expected_ctc" int,
+  "notice_period" varchar,
+  "resume" varchar,
+  "preferred_location" varchar,
+  "age" int
+);
+
+CREATE TABLE "user_employer" (
+  "id" SERIAL PRIMARY KEY,
+  "user_id" int,
+  "employer_id" int,
+  "is_current_employer" boolean
+);
+
+CREATE TABLE "company" (
+  "id" SERIAL PRIMARY KEY,
+  "name" varchar
+);
+
+CREATE TABLE "skill" (
+  "id" SERIAL PRIMARY KEY,
+  "name" varchar
+);
+
+CREATE TABLE "user_skill" (
+  "id" SERIAL PRIMARY KEY,
+  "user_id" int,
+  "skill_id" int,
+  "level" skill_level,
+  "last_used" varchar
+);
+
+CREATE TABLE "education" (
+  "id" SERIAL PRIMARY KEY,
+  "name" varchar
+);
+
+CREATE TABLE "user_education" (
+  "id" SERIAL PRIMARY KEY,
+  "user_id" int,
+  "employer_id" int,
+  "is_current_education" boolean
```

Comment:
Can we add two unique constraints on the following?

1. `(user_id, employer_id)`
2. `(user_id, is_current_education)`

Reply:
SQL will not be going to execute on a server it a just references for DBdiagram 

(the table will be created by Django Models)

Reply:
Yeah. I guessed the same too. I added the comments for completeness. :)